akka-actor/src/main/scala/akka/io/Tcp.scala

```diff
@@ -669,6 +670,25 @@ object TcpMessage {
     timeout:       FiniteDuration,
     pullMode:      Boolean): Command = Connect(remoteAddress, Option(localAddress), options, Option(timeout), pullMode)
 
+  /**
+   * The Connect message is sent to the TCP manager actor, which is obtained via
+   * [[TcpExt#getManager]]. Either the manager replies with a [[Tcp.CommandFailed]]
+   * or the actor handling the new connection replies with a [[Tcp.Connected]]
+   * message.
+   *
+   * @param remoteAddress is the address to connect to
+   * @param localAddress optionally specifies a specific address to bind to
+   * @param options Please refer to [[TcpSO]] for a list of all supported options.
+   * @param timeout is the desired connection timeout, `null` means "no timeout"
+   * @param pullMode enables pull based reading from the connection
+   */
+  def connect(
+    remoteAddress: InetSocketAddress,
+    localAddress:  InetSocketAddress,
+    options:       JIterable[SocketOption],
+    timeout:       java.time.Duration,
+    pullMode:      Boolean): Command = Connect(remoteAddress, Option(localAddress), options, Option(timeout.asScala), pullMode)
```

Comment:
I'd prefer calling the other method (even though the duplication looks harmless, it's still some duplication)
```
connect(remoteAddress, localAddress, options, timeout.asScala, pullMode)
```

Reply:
ok, it is also an option and i will do it